src/test/java/gameframework/game/GameUniverseDefaultTest.java

```diff
@@ -0,0 +1,126 @@
+package gameframework.game;
+
+import static org.junit.Assert.*;
+
+import gameframework.motion.blocking.MoveBlocker;
+import gameframework.motion.overlapping.Overlappable;
+
+import java.awt.Graphics;
+import java.awt.Point;
+import java.awt.Rectangle;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class GameUniverseDefaultTest {
+
+	protected GameUniverseDefaultImplBis game;
+	protected int cptGameEntityMoveBlocker;
+	protected int cptGameEntityOverlap;
+
+	@Before
+	public void init() {
+		this.game = new GameUniverseDefaultImplBis();
+		this.cptGameEntityMoveBlocker = 0;
+		this.cptGameEntityOverlap = 0;
+		this.game.setGameData(new GameData(new GameConfiguration()));
+	}
+
+	@Test
+	public void testAddGameEntity() {
+		this.game.addGameEntity(createGameEntityMoveBlocker());
+		assertSame(1, this.cptGameEntityMoveBlocker);
```

Comment:
the fact that 2 numbers with the same value are the same is an implementation detail of Java. Please use `assertEquals` instead